### PR TITLE
Fix assert for 'No Ceph tools pod found', was getting IndexError instead

### PR DESCRIPTION
**Pull request by jhutar**

### CODE REVIEW

File: ocs_ci/ocs/resources/pod.py [hunk 1]
```diff
@@ -262,11 +262,11 @@ def get_ceph_tools_pod():
...
         kind=constants.POD, namespace=config.ENV_DATA['cluster_namespace']
     )
-    ct_pod = ocp_pod_obj.get(
+    ct_pod_items = ocp_pod_obj.get(
         selector='app=rook-ceph-tools'
-    )['items'][0]
-    assert ct_pod, f"No Ceph tools pod found"
-    ceph_pod = Pod(**ct_pod)
+    )['items']
+    assert len(ct_pod_items) > 0, f"No Ceph tools pod found"
```
Comment (ebenahar) [inline ocs_ci/ocs/resources/pod.py]: You can also do:
```python 
assert ct_pod_items, "No Ceph tools pod found"
```

**Comment by clacroix12** [comment]
This looks like a good change. I would take a look at @ebenahar's comment and go that route as it's a bit cleaner. Also you need to sign off on your commits before this will be merge-able. Take a look here: https://github.com/red-hat-storage/ocs-ci/pull/457/checks?check_run_id=177113936

File: ocs_ci/ocs/resources/pod.py (hunk 1 above)
Comment (jhutar) [reply ocs_ci/ocs/resources/pod.py]: Done, thank you!

**Comment by jhutar** [comment]
@clacroix12 Should be OK now. Thank you!

**Pull request merged**